README.md

```diff
@@ -155,7 +155,7 @@ Name | Website | Region
 [Codeship](/company-profiles/codeship.md) | https://codeship.com/ | Worldwide
 [Codestunts](/company-profiles/codestunts.md) | https://codestunts.com/ | Worldwide
 [Cofense](/company-profiles/cofense.md) | https://cofense.com | USA
-[Coforma](/company-profiles/coforma.md) | https://coforma.io | USA
+[Cognizant](/company-profiles/cognizant.md) | https://www.cognizant.com/in/en | Worldwide
```

Comment:
Can you put the Coforma link back and add Cognizant as a new line under it please?

Reply:
ok sure my mistake sorry about that.